azure-pipelines.yml

```diff
@@ -282,8 +282,8 @@ stages:
         backwardCompatibleTestOptions: $(backwardCompatibleTestOptions_Linux_2_4)
         forwardCompatibleTestOptions: $(forwardCompatibleTestOptions_Linux_2_4)
     - version: '2.4.7'
-      enableForwardCompatibleTests: true
-      enableBackwardCompatibleTests: true
+      enableForwardCompatibleTests: false
+      enableBackwardCompatibleTests: false
```

Comment:
What happens if we enable these? I think John mentioned things should mostly work out of the box with pre .NET 6 compiled code as well, do we want to test that scenario incase customer code isn't updated?

Reply:
These tests fail because there are no available NuGets for it to test against.  Once we publish a release, we can enable these tests.